.vscode/launch.json

```diff
@@ -23,7 +23,7 @@
       "program": "${workspaceRoot}/scripts/debug-test.js",
       "cwd": "${fileDirname}",
       "stopOnEntry": false,
-      "args": ["-i", "--testPathPattern=${file}"],
+      "args": ["-i", "--testPathPattern=${fileBasenameNoExtension}"],
```

Comment:
This could be done in another pull request if y'all prefer.
I was the one who originally added this option but it only worked for me under linux. This update makes it work on Windows as well.